src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.tsx

```diff
@@ -175,35 +191,294 @@ type SingleBlastJobResultProps = {
   jobId: string;
   species: Species;
   diagramWidth: number;
+  blastDatabase: DatabaseType;
 };
 
+const hitsTableColumns: DataTableColumns = [
+  {
+    columnId: 'id',
+    title: 'ID',
+    isFilterable: false,
+    isHideable: false
+  },
+  {
+    width: '130px',
+    columnId: 'e_value',
+    title: 'E-value',
+    isSortable: true,
+    helpText: (
+      <span>
+        The number of times a match is expected to occur by chance <br></br>
```

Comment:
`<br></br>` is probably better represented in React as `<br />`?

Reply:
That's done